modules/preview-react/menu/stories/examples/Basic.tsx

```diff
@@ -57,7 +60,7 @@ export const Basic = () => {
           onClose must be set in order to the Menu to close properly after
           selecting a MenuItem
         */}
-        <Menu id={menuId} isOpen={isOpen} onClose={model.events.hide}>
+        <Menu id={menuId} isOpen={isOpen} onClose={model.events.hide} aria-labelledby={id}>
```

Comment:
In our docs above we called out using this prop for buttons but never added it to the example.